source/errors.ts

```diff
@@ -22,36 +49,34 @@ class GotError extends Error {
 			socketPath: options.socketPath,
 			protocol: options.protocol,
 			url: options.href,
-			gotOptions: options
+			gotOptions: options.options
 		});
 	}
 }
 
-module.exports.GotError = GotError;
-
-module.exports.CacheError = class extends GotError {
-	constructor(error, options) {
+export const CacheError = class extends GotError {
```

Comment:
Use the format `export class CacheError extends GotError {` on all of these instead.